best-practices/styling/index.html

```diff
@@ -1057,6 +1057,30 @@ <h3>Borders and lines</h3>
 							‘text-indent’ property</a></li>
 						</ul>
 					</details>
+					<div class="note">
+						<p>The same result may be achieved using the “border-image” approach
```

Comment:
I guess this won't work until the scripts are merged, right? All I get is an empty css example box in the preview and locally.

Reply:
Yes, sorry, I tested it locally by replacing

```js
script.src = "https://daisy.github.io/ebraille/best-practices/styling/handleBorderImageURLs.js";
```

with

```js
script.src = new URL("handleBorderImageURLs.js", document.baseURI).href;
```

Reply:
I was talking about the "Validate with alternative CSS" buttons. But now I noticed that the CSS is actually missing. Hum... I have no idea how that happened.

Reply:
Ya, I couldn't figure out if the example css was disappearing because some scripts were running on the file but not others. It might be easier to merge and then do a final check that everything is okay.

Reply:
This wasn't resolved yet. It seems like highlightjs is messing with the code blocks within `details` elements. I don't understand what is happening. For now my workaround is to not mark up these code blocks with `<pre><code>`. @mattgarrish Can you think of a way to fix this properly?